laplace/curvature/asdl.py

```diff
@@ -83,16 +83,23 @@ def closure():
                 )
                 return f
 
-            Ji, f = batch_gradient(
-                self.model,
-                closure,
-                return_outputs=True,
-                batch_size=self._get_batch_size(x),
-            )
+            Ji, f = batch_gradient(self.model, closure, return_outputs=True)
```

Comment:
Why is the `batch_size` argument no needed anymore, or perhaps, why was it even needed before?